pkg/apis/pipeline/v1/provenance.go

```diff
@@ -22,6 +24,8 @@ package v1
 type Provenance struct {
 	// ConfigSource identifies the source where a resource came from.
 	ConfigSource *ConfigSource `json:"configSource,omitempty"`
+
+	FeatureFlags *config.FeatureFlags `json:"featureFlags,omitempty"`
```

Comment:
do we need docstrings here?

Reply:
Ah yes! I moved it around. So I will add a docstring. Thanks for the catch.